universe/common/api/middleware/verify_user.go

```diff
@@ -7,27 +7,25 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
-	"github.com/momentum-xyz/ubercontroller/types/entry"
-	"github.com/momentum-xyz/ubercontroller/universe"
 	"github.com/momentum-xyz/ubercontroller/universe/common/api"
-	"github.com/momentum-xyz/ubercontroller/utils"
 )
 
+var secret *[]byte = nil
```

Comment:
you don't need pointer here, because slice is a pointer itself

Reply:
anyway let's move it to VerifyUser(), it's not a big deal to get secret few time while node loading